corehq/apps/ota/views.py

```diff
@@ -310,19 +310,20 @@ def _get_version_info(app_config):
             "latest_ccz_version": config.get_latest_app_version(build_profile_id),
         }
 
+    app = get_app_cached(domain, app_build_id)
```

Comment:
Can you confirm that this wont re-introduce the load @snopoke fixed recently when we released profile based updates?

Reply:
I have asked on the ticket for the same, he gave a green flag. 
Since we are using the cached version of app, I guess it should not be a problem.
But I would want @snopoke to have a look if there are any chances of things going bad again.

Reply:
this will indeed be an issue for redis, I've made a comment below about just using the app ID to check access instead of the whole app